libs/vm-modules/tests/unit/big_number_wrapper_tests.cpp

```diff
@@ -131,23 +131,27 @@ class UInt256Tests : public ::testing::Test
   static constexpr std::size_t    SIZE_IN_BITS  = 256;
   static constexpr std::size_t    SIZE_IN_BYTES = SIZE_IN_BITS / 8;
 
-  UInt256Wrapper zero{dummy_vm_ptr, dummy_typeid, 0};
+  UInt256Wrapper zero{
```

Comment:
How come this got reformatted

Reply:
I don't know, I don't bother/waste time with formating anymore and fully rely on `apply_style.py`.